src/conversion/model-to-view-converters.js

```diff
@@ -252,9 +254,11 @@ export function removeAttribute( attributeCreator ) {
  *
  * @param {module:engine/view/element~Element|Function} elementCreator View element, or function returning a view element, which will
  * be used for wrapping.
+ * @param {Function} [nameToConsumable] Optional function used to map conversion event name to consumable name. By default it
+ * uses {engine/conversion/model-to-view-converters~eventNameToConsumableType} function.
  * @returns {Function} Set/change attribute converter.
  */
-export function wrapItem( elementCreator ) {
+export function wrapItem( elementCreator, nameToConsumable = eventNameToConsumableType ) {
```

Comment:
Wow! You pass a function as a parameter to the function which is used as a listener... I know that functional programming is trendy, but it's too much for me, this part of the code is complicated enough without such parameters.

Is it because we `eventNameToConsumableType` does not handle markers events well? Can't we make `eventNameToConsumableType` smarter, then? Or maybe we should just copy (or extract) part of the code instead of hacking it...(*)